Gopkg.toml

```diff
@@ -0,0 +1,39 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/datasift/go-charset"
+  source = "github.com/stripe/go-charset"
+
+[[constraint]]
+  name = "github.com/elazarl/goproxy"
```

Comment:
This seems... wrong. If we're forking, we should't need to import that package at all.


At the very least, a `source` directive here pointing to the current repo? Not sure how dep would handle that though.